installation/cmd/run.sh

```diff
@@ -79,6 +83,16 @@ if [ -z "$KYMA_INSTALLATION" ]; then
   KYMA_INSTALLATION="minimal"
 fi
 
+if [[ ${DUMP_DB} ]]; then
+    sed -i - "s/image\:.*compass-schema-migrator.*/image\: compass-schema-migrator\:latest/" ${ROOT_PATH}/chart/compass/templates/migrator-job.yaml
```

Comment:
As a result of these few commands after the installation is done, we are left with a modified migrator-job.yaml file. If we only wanted to install compass on minikube with a db-dump once, in order to test something with "realish" data we would probably want the migrator-job's image to be left unchanged afterwards. The image should be different only untill the installation takes place. Probably we can save the image's name in a variable and change it back to the original one, once the installation is done.

Reply:
fixed